wradlib/util.py

```diff
@@ -584,26 +585,25 @@ def has_geos():
 
 
 def get_wradlib_data_path():
-    wrl_data_path = os.environ.get("WRADLIB_DATA", None)
-    if wrl_data_path is None:
-        raise OSError(
-            "`WRADLIB_DATA` environment variable not set. Please set `WRADLIB_DATA` "
-            "pointing to the location of `wradlib_data` repository on the filesystem."
+
+    if "WRADLIB_DATA" in os.environ:
+        wrl_data_path = os.environ["WRADLIB_DATA"]
+        warn(
+            "usage of the WRADLIB_DATA environment variable will be deprecated in the future in favor of the wradlid_data package",
+            FutureWarning,
         )
-    if not os.path.isdir(wrl_data_path):
-        raise OSError(f"`WRADLIB_DATA` path {wrl_data_path!r} does not exist.")
+    else:
+        wrl_data_path = DATASETS.abspath
+
+    if not os.path.exists(wrl_data_path):
+        os.makedirs(wrl_data_path)
+
     return wrl_data_path
 
 
 def get_wradlib_data_file(relfile):
     data_file = os.path.abspath(os.path.join(get_wradlib_data_path(), relfile))
```

Comment:
This will not work, if the wanted file isn't yet downloaded. We will have to explicitly 

```python
from wradlib_data import DATASETS
data_file = DATASETS.fetch(relfile)
```

to retrieve the file from the GH repo into a path which can be overridden by `WRADLIB_DATA`.

Reply:
This might be the root cause of the breaking tests.